runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -90,6 +91,20 @@
 		final JMenuItem openXpTracker = new JMenuItem("Open online tracker");
 		openXpTracker.addActionListener(e -> LinkBrowser.browse(XpPanel.buildXpTrackerUrl(client.getLocalPlayer(), skill)));
 
+		// Create skill icon panel for pause icon overlay
+		SkillIconPanel skillIconPanel = new SkillIconPanel(iconManager.getSkillImage(skill));
+
+		// Create pause menu
+		pause = new JMenuItem(UNPAUSED_STATE);
+		pause.addActionListener(e ->
+		{
+			boolean paused = pause.getText().equalsIgnoreCase(UNPAUSED_STATE);
```

Comment:
eeeech, really doing text comparison?

Reply:
I agree it's not my favorite thing to do. It seems robust however and considering the current code is structured in a somewhat different way I didn't want to alter that too much. Considering this works, is it really an issue?